public/router.php

```diff
@@ -25,15 +25,21 @@
 	date_default_timezone_set('UTC');
 }
 
-define('BASE_DIR', dirname(__DIR__).'/');
+$baseDir = str_replace('\\', '/', dirname(dirname(__FILE__)).'/');
+if (!is_file($baseDir.'pufferpanel')) {
+    $baseUrl = dirname($_SERVER['PHP_SELF']);
```

Comment:
Personally was the problem you don't want you source code in root of you're website. But you could fix this with nginx and apache. But that not a easy install. This way its just drop and go.